src/components/editor/property/PropertyResourceTemplate.jsx

```diff
@@ -6,12 +6,43 @@ import PropTypes from 'prop-types'
 import PropertyActionButtons from './PropertyActionButtons'
 import PropertyTemplateOutline from './PropertyTemplateOutline'
 
+/**
+ * Renders a sub-resource template (e.g. WorkTitle, WorkVariantTitle, TranscribedTitle)
+ * In Redux these are modeled like this:
+ *   http://id.loc.gov/ontologies/bibframe/title: {
+ *     'AE6Be-DJGj': {
+ *       'resourceTemplate:bf2:WorkTitle': {
+ *         'http://id.loc.gov/ontologies/bibframe/mainTitle: {
+ *
+ *         }
+ *       }
+ *     }
+ *     'Mxt-oGAg0s2': {
+ *       'resourceTemplate:bf2:WorkVariantTitle': {
+ *         'http://id.loc.gov/ontologies/bibframe/mainTitle: {
+ *
+ *         }
+ *       }
+ *     }
+ *     'dGMYKnhJhlG': {
+ *       'resourceTemplate:bf2:TranscribedTitle': {
+ *         'http://id.loc.gov/ontologies/bibframe/mainTitle: {
+ *
+ *         }
+ *       }
+ *     }
+ *   }
+ *
+ *  In the props we are passed the resourceTemplate, so we can get the rows from
+ *  the redux store by filtering path for items that have a key that matches
+ *  resourceTemplate.id
```

Comment:
thanks for this documentation - super helpful.

Reply:
+1